root/edit/details/SetCoverArt.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {Artwork} from '../../components/Artwork';
+import DescriptiveLink
+  from '../../static/scripts/common/components/DescriptiveLink';
+import ReleaseEvents
+  from '../../static/scripts/common/components/ReleaseEvents';
+import commaList from '../../static/scripts/common/i18n/commaList';
+
+type SetCoverArtEditT = {
+  ...EditT,
+  +display_data: {
+    +artwork: CompT<ArtworkT>,
+    +isOldArtworkAutomatic: boolean,
+    +release_group: ReleaseGroupT,
+  },
+};
+
+type Props = {
+  +edit: SetCoverArtEditT,
+};
+
+const SetCoverArt = ({edit}: Props): React.Element<'table'> => {
+  const display = edit.display_data;
+  const oldArt = display.artwork.old;
+  const newArt = display.artwork.new;
+
+  return (
+    <table className="details set-cover-art">
+      <tr>
+        <th>{addColonText(l('Release group'))}</th>
+        <td>
+          <DescriptiveLink entity={display.release_group} />
+        </td>
+      </tr>
+
+      <tr>
+        <th>{l('Old cover art:')}</th>
+        <td>
+          {oldArt ? (
+            <div className="editimage">
+              <div className="cover-image">
+                <Artwork artwork={oldArt} />
+              </div>
+              {oldArt.release ? (
+                <p>
+                  <DescriptiveLink entity={oldArt.release} />
+                  <br />
+                  <ReleaseEvents events={oldArt.release.events} />
+                </p>
+              ) : null}
+              <p>
+                {commaList(oldArt.types) || '-'}
+                <br />
+                {oldArt.comment}
+              </p>
+            </div>
+          ) : display.isOldArtworkAutomatic ? (
+            l(`The old image was selected automatically
+               from the earliest release in the release group.`)
+          ) : l(`We are unable to display this cover art.`)}
```

Comment:
_We_ (developers) should avoid to use _we_  (?) as some people seem to forget this is a community driven database:

```suggestion
          ) : l(`Unable to display this cover art.`)}
```

There currently are only 3 other occurrences of _We are_ and 20 other occurrences of _We_ in the server code.

Reply:
It sounds a bit too informal to me that way, because it's being kept in the active voice but missing a subject (so it's still implying "we were"/"we are").

We could write these in a passive voice, i.e. "This cover art couldn't/can't be displayed," I guess. Do you mean that "we" is too vague, or people can misinterpret it to mean the community?

Reply:
I mean people could misinterpret “we” as the website holders and mistakenly deduce that the database is maintained by a specific team rather than an open and welcoming community. This is probably more relevant to editors than to visitors, but we can probably follow it as a general rule.

Reply:
But it's not the community who has failed at showing the cover, it's the core team :D

Reply:
I did not write that. The issue is just that using a non-inclusive “we” in the website may let people think they are not part of the “million monkeys with typewriters” that maintain the database or that the code is not open source, which may let them think it is relevant to ask “to speak with [our] supervisor” or to threaten to “move legally” if not “all additions [are] returned”.

Reply:
TBH I just don't see how it can be read that way. We could change it to a passive voice, but the current string sounds the most natural to me.

Reply:
This is just following "We are unable to display history for this cover art". I'd keep this as is for now, if you really dislike our using "We" elsewhere you can always send a PR to change them all - but it seems like a bad idea to block what was (at least at the time) a regression fix because of a small wording issue.

Reply:
This discussion seems to have ended, so I'm going to merge this as it fixes a bug.  A ticket to remove "we" from our strings might make sense, though it shouldn't block this PR.